### PR TITLE
Disable targeted notifications in integration tests

### PR DESCRIPTION
**Pull request by sharwell**

### CODE REVIEW

File: eng/build.ps1 [hunk 1]
```diff
@@ -627,6 +627,13 @@ function Deploy-VsixViaTool() {
   # Configure RemoteHostOptions.OOP64Bit for testing
   $oop64bitValue = [int]$oop64bit.ToBool()
   &$vsRegEdit set "$vsDir" $hive HKCU "Roslyn\Internal\OnOff\Features" OOP64Bit dword $oop64bitValue
+
+  # Disable targeted notifications
+  if ($ci) {
+    # Currently does not work via vsregedit, so only apply this setting in CI
```
Comment (jasonmalinowski) [inline eng/build.ps1:633]: Is there a bug tracking this we can link to?

Comment (sharwell) [reply eng/build.ps1:633]: No, not sure it's a bug either

**Pull request merged**